src/Token/AccessToken.php

```diff
@@ -121,6 +121,10 @@ public function __construct(array $options = [])
             $expires = $options['expires'];
 
             if (!$this->isExpirationTimestamp($expires)) {
+                if (!is_numeric($expires)) {
+                    $expires = intval($expires);
```

Comment:
I'm not sure this check is useful. If something is not numeric, it usually won't have an intval worth using.

For example: https://3v4l.org/9r3Wj